tasking/serializers/submissions.py

```diff
@@ -4,15 +4,32 @@
 """
 from __future__ import unicode_literals
 
+from rest_framework import serializers
+
 from tasking.models import Submission
 from tasking.serializers.base import GenericForeignKeySerializer
+from tasking.common_tags import CANT_EDIT_TASK
 
 
 class SubmissionSerializer(GenericForeignKeySerializer):
     """
     Submission serializer class
     """
 
+    def validate_task(self, value):
+        """
+        Validate Task
+        """
+        if self.instance is not None:
+            if self.instance.task is value:
```

Comment:
Please use `==` (see: https://dbader.org/blog/difference-between-is-and-equals-in-python)